packages/fe-container/src/components/user-menu-overlay/UserMenuOverlay.less

```
@@ -0,0 +1,13 @@
+@import "../../styles/_variables";
+
+.userMenuOverlay_body {
+  box-shadow: @box-shadow-overlay;
```

```suggestion
  box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.15) !important;
```

This box shadow is not applied since ant overwrites it with class `.ant-menu-root.ant-menu-vertical`. Adding `!important` solves the issue.

Also, from mockup the box shadow to apply should be `box-shadow: 0px 2px 8px rgba(14, 14, 14, 0.15);`.